sdk/samples/aztables/README.md

```diff
@@ -0,0 +1,145 @@
+---
+page_type: sample
+languages:
+  - go
+products:
+  - azure
+  - aztables
```

Comment:
I think this 'product' is not just specific to Go, right? Do you know what the other languages are using for this tag?

Reply:
(it could very well be 'aztables', I just wanted to check)